components/PortalLink/PortalLink.js

```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link';
+
+import Arrow from '../../public/img/arrow.svg';
+
+export const PortalLink = (prop) => (
+  <Link href="/">
+    <a className="bg-accent-200 px-8 py-5 rounded-sm w-full md:mr-4 mb-4 sm:w-[calc(50%_-_16px)] lg:w-[calc(25%_-_16px)] transition-colors lg:hover:bg-light-50">
```

Comment:
`sm:w-[calc(50%_-_16px)] lg:w-[calc(25%_-_16px)]` looks like a very specific width calculation. Could you paste the design here as well? 

Try to keep the CSS as simple as possible. It will be just easier in the future.

Reply:
This calculates the width of the columns in flexbox. If you want to have two columns with same width with the gap between them, you need to set the column width to 50 %, but if you want to set the gap (f.e. margin-right: 16px), then the width of the column will be 50% + 16px and the second column will be on the new line. So you can set the width to the 50% - 16px and the margin-right to the 16px and it fits the container perfectly :)
It is similar for the 4-column layout as you can see in the `lg:w-[calc(25%_-_16px)]`

Reply:
I know about "gap", but If you care about IE due to grid, I don't know, if you want to use the gap :D